src/__tests__/card.test.ts

```diff
@@ -0,0 +1,101 @@
+import supertest from 'supertest';
```

Comment:
**suggestion:** I know it's quite common for putting tests in `__tests__` directory, but from my experience it's much more convenient to keep your tests in related domain directory. So in this case it would be `cards` directory. It will allow you to have much more control when your app scale, or when you want to split such domain into separate micro service.